sdk/rust/oak_tests/src/lib.rs

```diff
@@ -172,6 +173,10 @@ impl Drop for ChannelHalf {
 
 struct OakRuntime {
     termination_pending: bool,
+    // Map name of Wasm contents to a test entrypoint function pointer.
+    entrypoints: HashMap<String, NodeMain, RandomState>,
```

Comment:
Isn't `RandomState` the default?

https://doc.rust-lang.org/std/collections/struct.HashMap.html